velox/substrait/VeloxToSubstraitPlan.cpp

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/substrait/VeloxToSubstraitPlan.h"
+
+namespace facebook::velox::substrait {
+
+/// Velox Plan to Substrait
+::substrait::Plan* VeloxToSubstraitPlanConvertor::toSubstrait(
+    google::protobuf::Arena& arena,
+    const std::shared_ptr<const PlanNode>& vPlan) {
+  // Assume only accepts a single plan fragment
+  // TODO: convert the Split RootNode get from dispatcher to RootRel
```

Comment:
I do not understand this TODO. What is "Split RootNode"?

Reply:
sorry, remove this.